tests/test_environment.py

```diff
@@ -113,6 +113,8 @@ def equivalent_names(env1, env2):
                 print(f"{es_url} is properly {'blue' if is_blue else 'green'}.")
 
             env_pattern = full_test_env.replace("-", ".*")
+            if 'production' in env_pattern:
+                env_pattern = env_pattern.replace('production', 'prod')
```

Comment:
Lightly tested:
```suggestion
            env_pattern = (full_test_env
                           .replace("-", ".*")
                           .replace("production", "pro?d(uction)?")
                           .replace("development", "dev(elopment)?"))
```
The line for development is optional, but seemed symmetric. You could just do the first two replacements. But I think this is more general than the fix you have and likely to be more enduring.